src/lib/tls/asio/info.txt

```diff
@@ -0,0 +1,23 @@
+<defines>
+ASIO -> 20181218
```

Comment:
This should be something like `TLS_ASIO_STREAM` to distinguish it from other uses of asio